addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/handler/YeelightStripeHandler.java

```diff
@@ -0,0 +1,137 @@
+                    if (brightness.intValue() == 0) {
+                        DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(), DeviceAction.close);
+                    } else {
+                        if (mDevice.getDeviceStatus().isPowerOff()) {
+                            DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(), DeviceAction.open);
+                        }
+                        DeviceAction baction = DeviceAction.brightness;
+                        baction.putValue(brightness.intValue());
+                        DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(), baction);
+                    }
+                } else if (command instanceof OnOffType) {
+                    OnOffType onoff = (OnOffType) command;
+                    DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(),
+                            onoff == OnOffType.ON ? DeviceAction.open : DeviceAction.close);
+                } else if (command instanceof IncreaseDecreaseType) {
+                    IncreaseDecreaseType increaseDecrease = (IncreaseDecreaseType) command;
+                    DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(),
+                            increaseDecrease == IncreaseDecreaseType.INCREASE ? DeviceAction.increase_bright
+                                    : DeviceAction.decrease_bright);
+                }
+                break;
+
+            case YeelightBindingConstants.CHANNEL_COLOR:
+                if (command instanceof HSBType) {
+                    HSBType color = (HSBType) command;
+                    DeviceAction caction = DeviceAction.color;
+                    caction.putValue(color.getRGB() & 0xFFFFFF);
+                    DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(), caction);
+                }
+                break;
+            case YeelightBindingConstants.CHANNEL_COLOR_TEMPERATURE:
+                if (command instanceof PercentType) {
+                    PercentType ct = (PercentType) command;
+                    DeviceAction ctaction = DeviceAction.colortemperature;
+                    ctaction.putValue(48 * ct.intValue() + 1700);
+                    DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(), ctaction);
+                } else if (command instanceof IncreaseDecreaseType) {
+                    IncreaseDecreaseType increaseDecrease = (IncreaseDecreaseType) command;
+                    DeviceManager.getInstance().doAction(channelUID.getThingUID().getId(),
+                            increaseDecrease == IncreaseDecreaseType.INCREASE ? DeviceAction.increase_ct
+                                    : DeviceAction.decrease_ct);
+                }
+                break;
+
+            default:
+                break;
+        }
+    }
+
+    @Override
+    public void initialize() {
+        mLogger.info(TAG + ": initialize");
```

Comment:
Please use parametrized list instead of string concatenation. 
Also, please change `info` to `debug`